csrankings-a.csv

```diff
@@ -657,6 +657,7 @@ Alexander Binder,SUTD,https://istd.sutd.edu.sg/people/faculty/alexander-binder,5
 Alexander Borgida,Rutgers University,https://www.cs.rutgers.edu/~borgida,S5AaMfgAAAAJ
 Alexander Brodsky 0001,George Mason University,https://cs.gmu.edu/~brodsky,NOSCHOLARPAGE
 Alexander Brodsky 0003,Dalhousie University,https://www.dal.ca/faculty/computerscience/faculty-staff/alex-brodksy.html,NOSCHOLARPAGE
+Alexander C. Berg,Univ. of California - Irvine,http://acberg.com/,jjEht8wAAAAJ

```

Comment:
Must be >= 75% appointed over the year at UC Irvine to be eligible for inclusion. Please confirm. (Many FAIR scientists do not meet this criteria.)